01 Operator_Overloading/01 OperatorOverloading/OperatorOverloading.Model/Money.cs

```diff
@@ -50,6 +51,37 @@ public Money(double amount, string currency)
      
         }
 
+        public Money Convert(string targetCurrency)
+        {
+            double rate;
+            // currency value validation here:
+
+            if (string.IsNullOrWhiteSpace(targetCurrency) == true && targetCurrency.Length != 3)
+            {
+                CustomExceptions.ExceptionThrower.ThrowImproperValueForCurrency();
+            }
+            if (this.Currency.Equals("USD") == false && targetCurrency.ToUpper().Equals("USD") == false)
```

Comment:
Why is their a check for USD? Your Convert function should work for all currencies.